kellyramirezz/Resource_Services

kellyramirezz opened the pull request:
The font doesn't seem to be the same, even though I got it from Figma. With the margins and padding, do I make them up myself or is there a way to see it on Figma? And I wasn't sure I was supposed to do just the Red text or more.

VacaTheCow commented on src/app/Services/ResourceServices.tsx:
```diff
@@ -0,0 +1,25 @@
+const ResourceServices = () => {
+  return (
+    <div className="relative h-screen w-full">
+      <div className="h-971px w-1116px absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 transform rounded-[40px] border-[4px] border-[#93B48F] bg-[#FFFBF9] text-center font-[Besley] text-[24px]">
```
Change all hard-coded values

VacaTheCow commented on src/app/Services/ResourceServices.tsx:
```diff
@@ -0,0 +1,25 @@
+            UCR's Health Promotion and Education Department
+          </div>
+          <div className="font-[400] text-[#442F2C]">
+            <div className="">
```
remove unnecessary div and classname

VacaTheCow commented on src/app/Services/ResourceServices.tsx:
```diff
@@ -0,0 +1,25 @@
+const ResourceServices = () => {
+  return (
+    <div className="relative h-screen w-full">
+      <div className="h-971px w-1116px absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 transform rounded-[40px] border-[4px] border-[#93B48F] bg-[#FFFBF9] text-center font-[Besley] text-[24px]">
```
use a tailwind preset "rounded", don't hard code

<img width="104" alt="image" src="https://github.com/user-attachments/assets/27bde376-aee0-484f-a411-8d5088240623" />

VacaTheCow replied:
And for the font issue, I have someone else importing the exact font so when thats approved I'll have you fix that later here.

VacaTheCow replied:
This is how it currently looks:
<img width="1443" alt="image" src="https://github.com/user-attachments/assets/099036a7-43ed-4a8a-9530-98b785bd29d7" />

But it has to look like this:
<img width="833" alt="image" src="https://github.com/user-attachments/assets/c67b0db1-2f48-4b06-874a-2c106f5dbabb" />
And fit almost the ENTIRE width of the screen

VacaTheCow reviewed (changes requested):
Let's avoid hard-coding the exact width and heights, I know you didn't go to our 2 meeting but the reason we want to avoid doing this is because h-971px might look good on your laptop, but what happens when I render the webpage on my small phone screen? It might look a bit funky. So use a preset tailwind [width](https://tailwindcss.com/docs/width) and [height](https://tailwindcss.com/docs/height).

kellyramirezz commented:
![Screenshot 2025-04-28 012614](https://github.com/user-attachments/assets/58844fc0-f453-49ec-ba93-69550e7a8373)
I updated and implemented the feedback. The only thing is I'm not 100% on the height.

VacaTheCow commented:
Good job!
